### ISSUE TITLE
Details for Training on deep network

### ISSUE THREAD

**Issue by AlexHu123** [shaohua0116/Group-Normalization-Tensorflow#5]
Hi, shaohua! 
How are the results trained on deep neural networks?
Recently I tried to train on 3D ConvNet, but I am confused that the training accuracy gets high(the class score are going to be the same).
